src/app/shared/main-navigation/mat-main-nav/menu-items.const.ts

```diff
@@ -93,17 +93,17 @@ export const menuItems: IMenuItem[] = JSON.parse(
                 "description": ""
             },
             {
-                "type": 0,
-                "route": "/concerns",
+                "type": 1,
+                "route": "concerns/",
                 "name": "Concerns",
                 "description": ""
             },
             {
-                "type": 0,
-                "route": "/connections",
+                "type": 1,
+                "route": "admin/manage-connections/create",
```

Comment:
Will these direct to the different domain that the legacy apps are on?

Reply:
Yeah, having dug through the code, "type":1 points the link to tis (preprod and prod)